generator-scrivito/generators/obj/index.js

```diff
@@ -0,0 +1,98 @@
+const Generator = require("yeoman-generator");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.log("Welcome in scrivito Obj/Widgets generator!");
+  }
+
+  start() {
+    this._creatingObj();
+  }
+
+  _creatingName(name) {
+    const defaultName = name;
+    return defaultName;
+  }
+
+  _creatingUpperName(name) {
+    const defNameUpper = name.charAt(0).toUpperCase() + name.slice(1);
+    return defNameUpper;
+  }
+
+  _creatingObj() {
```

Comment:
`_creatingObj` is currently only used in `start`. Please inline the content of `_createObj` into `start`.